docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -1409,9 +1421,11 @@ VertxDynamicGraphQLClient client = builder.build();
 
 include::{generated-dir}/config/quarkus-oidc-client.adoc[opts=optional, leveloffset=+1]
 
+ifndef::no-quarkus-oidc-token-propagation-reactive[]
```

Comment:
Maybe we did agree, sorry if I don't recall it, but it appears counter intuitive to me that we start blocking the content for the extensions which are making its way to the product, even if they are not fully supported yet

Reply:
Unfortunately, if I recall the discussion correctly, the decision was to conditionalize the content until they are eventually productized (in 3.14?).